addons/web/static/src/core/domain_selector/domain_selector_operator_editor.js

```diff
@@ -37,7 +37,7 @@ export function getDomainDisplayedOperators(fieldDef) {
             ];
         case "date":
         case "datetime":
-            return ["=", "!=", ">", ">=", "<", "<=", "between", "within", "set", "not_set"];
+            return ["=", "!=", ">", ">=", "<", "<=", "between", "is_not_between", "within", "is_not_within", "set", "not_set"];
```

Comment:
We don't have to add them, for the purposes of this task.

But since "not" is not automatically distributed on "complex" domains you can only get to these by typing them out entirely so they'd be low in discoverability